documentation/source/user_section/installation/windows.rst

```diff
@@ -86,18 +117,57 @@ Windows Subsystem for Linux (WSL) is available on Windows 10 and it makes it pos
       cd /mnt/c
 
 
+WSL Installation with GUI (Optional)
+************************************
+
+If you would like to use SCT's GUI features, or if you would like to try FSLeyes within the same Ubuntu environment, first complete the previous "Basic Installation" section, then continue on to the steps below.
+
+#. Download and install `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_, a program that makes it possible to run Linux GUI programs installed with WSL.
+
+#. Run the newly installed ``XLaunch`` program, then click the following settings:
+
+   - On the "Display settings" page, click "Next".
+   - On the "Client startup" page, click "Next".
+   - On the "Extra settings" page, check the "Disable access control" box, then click "Next".
+   - Click "Finish", then click "Allow access" when prompted by Windows Firewall.
+   - You should now see the X Server icon running in the bottom-right system tray in your taskbar.
+
+#. Next, run the following commands in Ubuntu, depending on the version of WSL you are using.
+
+   WSL1:
+
+   .. code::
+
+      echo "export DISPLAY=localhost:0.0" >> ~/.bashrc
+      echo "export LIBGL_ALWAYS_INDIRECT=0" >> ~/.bashrc
+      source ~/.bashrc
+
+   WSL2:
+
+   .. code::
+
+      echo "export DISPLAY=$(awk '/nameserver / {print $2; exit}' /etc/resolv.conf 2>/dev/null):0.0" >> ~/.bashrc
+      echo "export LIBGL_ALWAYS_INDIRECT=0" >> ~/.bashrc
+      source ~/.bashrc
+
+#. Finally, run the ``sct_check_dependencies`` command in Ubuntu to verify that matplotlib and PyQt figures can be opened by SCT.
+
+#. Optionally, you can install FSLeyes using the following commands:
+
+   .. code::
+
+      source ${SCT_DIR}/python/etc/profile.d/conda.sh
+      conda create -c conda-forge -p ~/fsleyes_env fsleyes -y
+      sudo ln -s ~/fsleyes_env/bin/fsleyes /usr/local/bin/fsleyes
+
+   These instructions will install FSLeyes into a fresh ``conda`` environment, then create a link to FSLeyes so that you can use the ``fsleyes`` command without having to activate the conda environment each time.
+
+
 Option 2: Install with Docker
 -----------------------------
 
 `Docker <https://www.docker.com/what-container>`_ is a portable (Linux, macOS, Windows) container platform.
 
-In the context of SCT, it can be used:
-
-- To run SCT on Windows, until SCT can run natively there
-- For development testing of SCT, faster than running a full-fledged
-  virtual machine
-- <your reason here>
-
 Basic Installation (No GUI)
 ***************************
 
```

Comment:
should we say something like "Below we show you two examples of Docker config (pick one)".

to make sure people won't actually do both sequentially